README.md

```diff
@@ -80,88 +80,86 @@ interface CheckerInfoMessage {
+    dnsSuffix: string | null;                   // The DNS suffix (including the leading dot), if DNS is used. Example: ".bambi.ovh"
     services: Service[];
     teams: Team[];
 }
 
 interface Team {
-    teamName: string;                           // "teamname40"
-    teamId: number;                             // 40
-    totalPoints: number;                        // 2692.662622758371
-    attackPoints: number;                       // 0.0
-    lostDefensePoints: number;                  // 0.0
-    serviceLevelAgreementPoints: number;        // 2692.662622758371
+    teamName: string;                           // The name of the team.
+    teamId: number;                             // The id of the team.
+    totalPoints: number;                        // The total points of the team.
+    attackPoints: number;                       // The attack points of the team.
+    defensePoints: number;                      // The defense points of the team.
+    serviceLevelAgreementPoints: number;        // The SLA points of the team.
     serviceDetails: ServiceDetail[];
 }
 
 interface ServiceDetail {
-    serviceId: number;                          // 0.0
-    attackPoints: number;                       // 0.0
-    lostDefensePoints: number;                  // 0.0
-    serviceLevelAgreementPoints: number;        // 0.0
-    serviceStatus: string;                      // INTERNAL_ERROR,OFFLINE,MUMBLE,RECOVERING,OK,INACTIVE
+    serviceId: number;                          // The id of the service.
+    attackPoints: number;                       // The attack points of the team in the service.
+    defensePoints: number;                      // The defense points of the team.
+    serviceLevelAgreementPoints: number;        // The SLA points of the team in the service.
+    serviceStatus: string;                      // "INTERNAL_ERROR", "OFFLINE", "MUMBLE", "RECOVERING", "OK", "INACTIVE"
     message: string | null;                     // Leave null for no message, otherwise the message is displayed
 }
 
 interface Service {
-    serviceId: number;
-    serviceName: string;
-    maxStores: number;
+    serviceId: number;                          // The id of the service.
+    serviceName: string;                        // The name of the service.
+    flagVariants: number;                       // The amount of different flag variants.
     firstBloods: FirstBlood[];
 }
 
 interface FirstBlood {
-    teamId: number;
-    timestamp: string;                          // Timestamps according ISO-86-01 ("yyyy-MM-ddTHH:mm:ss.fffZ")
-    timeEpoch: number;                          // Unix time in seconds
-    roundId: number;                            // 1
-    storeDescription: string | null;            // "Private user notes"
-    storeIndex: number;                         // 0
+    teamId: number;                             // The id of the team that scored the firstblood.
```

Comment:
could add teamName here too, for ease of use

Reply:
could be nice for custom parsers from teams, I guess